src/auth.js

```diff
@@ -52,6 +52,14 @@ function * createUserWithEmailAndPassword (email, password) {
   return yield call([auth, auth.createUserWithEmailAndPassword], email, password)
 }
 
+function * sendEmailVerification (currentUser) {
+  return yield call(currentUser.sendEmailVerification)
+}
+
+function * updatePassword (currentUser, password) {
```

Comment:
Same, this saga should be able to take `password` as it's only parameter and call directly `auth.currentUser.updatePassword`.
Might be worth moving this to a different PR. ;)